web-ng/src/app/components/map/map.component.ts

```diff
@@ -219,6 +232,9 @@ export class MapComponent implements AfterViewInit, OnDestroy {
       this.selectMarker(marker);
     }
     this.markers.push(marker);
+    if (selectedFeatureId === feature.id) {
```

Comment:
Is the intended to handle the case where the features haven't loaded yet, but `f=..` is already in the URL? If so, please clarify with a comment.

If that's the case, I think it should be safe to add a non-async method getFeatureId() the NavigatorService so that we don't need to pass this in as a streamed arg.

Reply:
Moved this up in onProjectAndFeaturesUpdate.